scripts/CsvToJson.js

```diff
@@ -34,10 +39,12 @@ class CsvToJsonScript extends Base {
 
     // Execute external CSV to JSON module
     exec(
-      `${csvToJsonPath} ${sourceFilepath} > ${outputFilepath}`,
+      `${csvToJsonPath} ${sourceFilepath} > ${outputFilepath} ${moduleParams}`,
       (err, stdout, stderr) => {
         if (err) throw err;
-        console.log(`stdout: ${stdout}`, `stderr: ${stderr}`);
+        if (stdout) console.log(`stdout: ${stdout}`);
+        if (stderr) console.log(`stderr: ${stderr}`);
```

Comment:
I would `throw` the error the the console. The reason for that is that if I run this programatically and listen to the exit code, this will silently fail.

What is being returned if the script succeeds tho? You might want to return the path to the file there, because for e.g. in iTerm you can configure extensions so that you can just cmd + click to open a file from the terminal directly.

Reply:
So we're already throwing `if (err)`.. shall we do the same if `stderr`?

Reply:
I like the idea of outputting the path to the file on success, implementing that now 👌

Reply:
Regarding the stdout/stderr, was sorta just following the Node.js docs here: https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback

Let me know if you think there's an optimal way to handle 👍